From hackermail Thu Jan  1 00:00:00 1970
From: SeongJae Park <sj@kernel.org>
To: Andrew Morton <akpm@linux-foundation.org>
Cc: SeongJae Park <sj@kernel.org>
Cc: damon@lists.linux.dev
Cc: kernel-team@meta.com
Cc: linux-kernel@vger.kernel.org
Cc: linux-mm@kvack.org
Date: Tue, 15 Jul 2025 10:06:38 -0700
Subject: [PATCH mm-unstable 1/2] mm/damon: add kernel-doc comment for
         damon_operations->cleanup_target

Commit 769bd386e490 ("mm/damon/core: add cleanup_target() ops callback")
on mm-unstable added cleanup_target field to damon_operations, but
forgot adding the kernel-doc description.  This causes a build warning.
Fix it by adding the description.

Fixes: 769bd386e490 ("mm/damon/core: add cleanup_target() ops callback") # mm-unstable
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Closes: https://lore.kernel.org/20250715185549.264260b8@canb.auug.org.au
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 include/linux/damon.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/damon.h b/include/linux/damon.h
index 34fc5407f98e..efabc3ba52ee 100644
--- a/include/linux/damon.h
+++ b/include/linux/damon.h
@@ -576,6 +576,7 @@ enum damon_ops_id {
  * @get_scheme_score:		Get the score of a region for a scheme.
  * @apply_scheme:		Apply a DAMON-based operation scheme.
  * @target_valid:		Determine if the target is valid.
+ * @cleanup_target:		Clean up each target before deallocation.
  * @cleanup:			Clean up the context.
  *
  * DAMON can be extended for various address spaces and usages.  For this,
-- 
2.39.5
